dev/dev-env-setup.sh

```diff
@@ -0,0 +1,233 @@
+#!/usr/bin/env bash
```

Comment:
Does this shebang `#!/usr/bin/env bash` support windows system ?
Looks like windows path is something like `C:\....`

Reply:
Oh maybe I am wrong, I think the `bash` part means the one used in windows ？